webpack.common.js

```diff
@@ -18,8 +18,6 @@ const versionHash = md5(appVersion).substr(0, 7)
 const SCOPE_VERSION = JSON.stringify(versionHash)
 const ICONFONT_NAME = `iconfont-vue-${versionHash}`
 
-console.info('This build version hash is', versionHash, '\n')
```

Comment:
No, please let this here

Reply:
That is just for the bundle-analyzer. Regarding this PR, just have a look onto the last commit https://github.com/nextcloud/nextcloud-vue/commit/d82e0952528961d3b180cad4670d85cb57c4f3fc. Everything else would be removed otherwise.

Reply:
Just pushed a stripped version. So it's easier to look into.